Dockerfile

```diff
@@ -24,9 +24,11 @@ RUN pip install --upgrade --no-cache-dir hokusai
 RUN gem install bundler -v '<2' && \
     bundle config --global frozen 1
 
-# Set up gems
-COPY Gemfile Gemfile.lock .ruby-version ./
-RUN bundle install -j4
+# Set up gems & packages, empty cache to save space
+COPY Gemfile Gemfile.lock .ruby-version package.json yarn.lock ./
+RUN bundle install -j4 && \
+    yarn install --frozen-lockfile --quiet && \
+    yarn cache clean --force
```

Comment:
Splitting up the ruby dependency step from the yarn step might better utilize docker caching. E.g., a change to Gemfile should force just a re-bundling. Or (possibly better), a change to `package.json` shouldn't force gems to be reinstalled.

(Not a big deal in this project, but I'm anticipating that these will get copied elsewhere.)